chief_file_server.py

```diff
@@ -0,0 +1,50 @@
+from http.server import BaseHTTPRequestHandler, HTTPServer
+import socket
+import sys
+import importlib.util
+import random
+
+class ChiefFileServer(BaseHTTPRequestHandler):
+    def do_POST(self):
+        
+        def upload_result():
+            # to do
+            # merge result
+            # send fragment
+            fragment = next(splitter_generator)
+            id = str(random.randint(0,1e+10))
+            fragname = "fragment"+id
+
+            with open("/app/Data/fragment_cache/"+fragname+".txt", "w") as file:
+                file.write(str(fragment))
+
+        endpoints = {"/upload/result/":upload_result}
```

Comment:
Maybe I'm tripping but I think theres some python fuckery here? This line is indented one too many and is part of the `upload_result` function.